### PR TITLE
`.era` store docs

### PR DESCRIPTION
**Pull request by arnetheduck**
Initial documentation for era file generation and usage

### CODE REVIEW

**Review by tersec — APPROVED** [review]

File: docs/the_nimbus_book/src/era-store.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Era store
+
+!!! warning
+    This feature is currently in BETA - nodes using era files may need to be resynced as the data format is not yet considered stable.
+
+Era files are a long-term archival format for Ethereum data. They are used to provide an easy interchange medium that clients interested in deep ethereum history can use to recreate past state.
```
Comment (zah) [inline docs/the_nimbus_book/src/era-store.md]:
```suggestion
Era files are a long-term archival format for Ethereum data. They are used to provide an easy interchange medium that clients interested in deep ethereum history can use to recreate past states.
```

File: docs/the_nimbus_book/src/era-store.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+## Options
+
+You can pass a custom era store location to Nimbus using `--era-dir` - multiple Nimbus instances can share the same era store.
+
+```sh
+nimbus_beacon_node --era-dir:/path/to/era
+```
+
+!!! tip
+    Multiple nimbus beacon node instances can share the same era store
```
Comment (zah) [inline docs/the_nimbus_book/src/era-store.md:75]: This sentence is duplicated just couple of lines above.

File: docs/the_nimbus_book/src/era-store.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+# Create a directory for the era store
+mkdir -p era
+cd era
+
+# Launch the era export
+../../../ncli_db exportEra --db:../db
+```
+
+The first time the export is run, full history is exported which make take some time. Subsequent runs will top up the era store with new blocks.
```
Comment (kdeme) [inline docs/the_nimbus_book/src/era-store.md]:
```suggestion
The first time the export is run, full history is exported which may take some time. Subsequent runs will top up the era store with new blocks.
```

File: docs/the_nimbus_book/src/era-store.md (hunk 1 above)
Comment (kdeme) [inline docs/the_nimbus_book/src/era-store.md]:
```suggestion
Era files are a long-term archival format for Ethereum data. They are used to provide an easy interchange medium that clients interested in deep Ethereum history can use to recreate past state.
```

**Pull request merged**